MeTTa-tasks.metta

```diff
@@ -0,0 +1,56 @@
+(= (leap-year $x)
+    (if (== (% $x 4) 0)
+        (if (== (% $x 100) 0)
+            (if (== (% $x 400) 0)
+                True
+                False
+            )
+        True
+    )
+    False
+)
+)
+! (leap-year 1996)
+! (leap-year 2000)
+! (leap-year 1600)
+! (leap-year 200)
+! (leap-year 400)
+; End of function leap-year
+
+; Planetary age conversion
+(: Mercury Planet)
+(: Venus Planet)
+(: Earth Planet)
+(: Mars Planet)
+(: Jupiter Planet)
+(: Saturn Planet)
+(: Uranus Planet)
+(: Neptune Planet)
+
+(: EarthYears (-> (Planet Number) Number))
+
+(Mercury 0.2408467)
+(Venus 0.61519726)
+(Earth 1)
+(Mars 1.8808158)
+(Jupiter 11.862615)
+(Saturn 29.447498)
+(Uranus 84.016846)
+(Neptune 164.79132)
+
+(= (EarthYears ($planet $seconds))
+    (match &self ($planet $x) (println! (Person is(* (/ $seconds 31557600) $x) Earth years old)))
+    )
```

Comment:
@Kirubel-the-student that's not how we found years for a given planet and its second. we need to find the amount of second it take in terms of earth for a given another planet.

Reply:
I came to you about this problem Yeabsira. I had an issue with the way the problem was framed. For me, it didn't make much sense to convert from seconds on one planet to seconds on another because the way time is measured doesn't depend from planet to planet. Days could be long or shorter depending on how farther or closer the particular planet is to the sun relative to Earth, but a second here is the same second on Mercury or Mars.

Anyway, I was just trying to show how I can apply pattern matching to solve the problem. I didn't intend to get the numbers correct because of the above issue. 
I know it doesn't matter a bit but will you think about that?